charter.html

```diff
@@ -530,7 +530,8 @@ <h3 id=notices>Announcements</h3>
 Community Group GitHub repositories</a>. All issues and pull requests
 posted to those repositories are automatically sent to the Community
 Group's mailing list, public-patcg@w3.org, that all members are subscribed
-to upon joining the Community Group.
+to upon joining the Community Group; refer to the
+<a href="https://patcg.github.io/">PATCG homepage</a> for information to join.
```

Comment:
I think we should send them to the Join button at https://www.w3.org/community/patcg/ ?
```suggestion
<a href="https://www.w3.org/community/patcg/" target="_blank">PATCG community page</a> for information to join.
```